data/aliases.js

```diff
@@ -201,8 +201,13 @@ exports.BattleAliases = {
 	"floetteeternalflower": "Floette-Eternal",
 	"ashgreninja": "Greninja-Ash",
 	"zydog": "Zygarde-10%",
+	"zydoge": "Zygarde-10%",
+	"zygardedog": "Zygarde-10%",
 	"zygarde50": "Zygarde",
+	"zyc": "Zygarde-Complete",
 	"zygarde100": "Zygarde-Complete",
+	"zygardec": "Zygarde-Complete",
+	"zygod": "Zygarde-Complete",
```

Comment:
Apparently in OM it's used quite a bit

[21:55:16] partys over ?_: ive heard zygod for complete
[21:55:31] @Ridaz: where
[21:55:37] partys over ?_: friends that play bh
[21:55:38] Macch Aeger: it's pretty common in the om room
[21:56:31] +ih8ih8sn0w: zygod is pretty common in oms, yeah (and ive probably tried to do !dt zygod on a couple occasions)